lichess-bot.py

```diff
@@ -532,24 +533,31 @@ def score_to_wdl(score):
                     score = random_move["score"]
                     move = random_move["uci"]
                     logger.info("Got move {} from chessdb.cn (wdl: {})".format(move, score_to_wdl(score)))
-                    return move
+                    return move, score_to_wdl(score)
     except Exception:
         pass
 
-    return None
+    return None, None
 
 
-def get_online_move(li, board, game, online_moves_cfg):
+def get_online_move(li, board, game, online_moves_cfg, draw_or_resign_cfg):
     online_egtb_cfg = online_moves_cfg.get("online_egtb", {})
     chessdb_cfg = online_moves_cfg.get("chessdb_book", {})
     lichess_cloud_cfg = online_moves_cfg.get("lichess_cloud_analysis", {})
-    best_move = get_online_egtb_move(li, board, game, online_egtb_cfg)
+    offer_draw = False
+    resign = False
+    best_move, wdl = get_online_egtb_move(li, board, game, online_egtb_cfg)
     if best_move is None:
         best_move = get_chessdb_move(li, board, game, chessdb_cfg)
+    else:
+        if draw_or_resign_cfg.get('offer_draw_enabled', False) and abs(wdl) <= draw_or_resign_cfg.get('offer_draw_egtb_score', 0):
+            offer_draw = True
+        if draw_or_resign_cfg.get('resign_enabled', False) and wdl == -2:
+            resign = True
```

Comment:
I would add a blank line after this line (before `if best_move is None:`, since it's not really a part of the next if-block structure.